src/libLookup/Lookup.cpp

```diff
@@ -782,7 +782,8 @@ void Lookup::RetrieveDSBlocks(vector<DSBlock>& dsBlocks, uint64_t& lowBlockNum,
     lowBlockNum = 1;
```

Comment:
lowBlockNum = 1 and partialRetrieve = true, should return only the latest block ?